### PR TITLE
Dart implementation of Casbin

### PR DESCRIPTION
**Pull request by KNawm**
[WIP]

### CODE REVIEW

**Comment by romitkarmakar** [comment]
@hsluoyz I think this PR is a good implementation of Casbin. I am looking forward to work in this project through this PR.

**Comment by hsluoyz** [comment]
@nodece @GopherJ

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```
Comment (nodece) [inline CHANGELOG.md:1]: I suggestion use v0.0.1 as first version.

**Comment by nodece** [comment]
lgtm.

File: CHANGELOG.md (hunk 1 above)
Comment (hsluoyz) [inline CHANGELOG.md:1]: 0.0.1

**Comment by hsluoyz** [comment]
I will approve this PR for now. But please setup test cases and CI ASAP. So we can know whether this code actually works as expected.

**Pull request merged**